src/components/breadcrumbs/Breadcrumbs.vue

```diff
@@ -0,0 +1,80 @@
+<template>
```

Comment:
Would suggest different name for this component, or split breadcrumbs and collapse component.

Breadcrumbs have nothing to do with sidebar collapse.